pkg/config/config.go

```diff
@@ -94,7 +94,12 @@ func init() {
 
 	} else {
 		Datadog.SetDefault("container_proc_root", "/proc")
-		Datadog.SetDefault("container_cgroup_root", "/sys/fs/cgroup/")
+		// for amazon linux the cgroup directory on host is /cgroup/
+		if _, err := os.Stat("/cgroup/"); err == nil {
```

Comment:
Let's be more specific on the Stat and make sure we don't have an empty folder. We could look for `/cgroup/memory/memory.stat` for example